pkg/geo/geos/geos.go

```diff
@@ -983,3 +998,22 @@ func Node(a geopb.EWKB) (geopb.EWKB, error) {
 	}
 	return cStringToSafeGoBytes(cEWKB), nil
 }
+
+// VoronoiDiagram Computes the Voronoi Diagram from the vertices of the supplied EWKBs.
+func VoronoiDiagram(a, env geopb.EWKB, tolerance float64, onlyEdges bool) (geopb.EWKB, error) {
```

Comment:
can you rename `a` to `g`?

Reply:
for the shake of familiarity, I've avoided just this one as there is already `g, err := ensureInitInternal()` in line 1004.